domains/albakhet.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "my email forwarding domain",
+  "owner": {
+    "username": "Somaiya-XI",
+    "discord": "Sen!#0577"
+  },
+  "record": {
+    "MX": [
+      {"priority": 10, "value": "mx1.improvmx.com"},

```

Comment:
We do not have priority properties for MX records. See docs: https://is-a.dev/docs/domain-structure/